docs/GE_Checklist.md

```diff
@@ -6,40 +6,151 @@ to be taken in order to publish a new FIWARE GE.
 Any doubt or request to the FIWARE Foundation shall be made by sending an email
 to [fiware-help@lists.fiware.org](mailto:fiware-help@lists.fiware.org)
 
--   New entry in the [FIWARE Catalogue](https://catalogue-server.fiware.org)
-    under the corresponding Chapter agreed at TSC Level. See also
+### New Generic Enablers
+
+The following mimimal requirements must be met before a new contribution can be
+considered as an Incubated Generic Enabler. The acceptance can take time, for
+instance to allow studying the quality of the product, internal architecture,
+software design, code cleanliness etc.
+
+-   All New Generic Enablers **MUST** be presented to the TSC for candidature
+    and accepted into an existing Chapter.
+-   A New Generic Enabler **MUST** show how it integrates within the overall
+    FIWARE Architecture
+-   The codebase **MUST** be available on [GitHub](https://github.com)
+-   The GitHub repository **MUST** include an appropriate Open Source License
+-   For functional readiness at a minimum, a new incubated enabler **MUST** be
+    able to be instantiated directly in a running state by any competent
+    developer without the need to refer to other dependent technologies - for
+    example:
+
+    -   [Keyrock](https://fiware-idm.readthedocs.io/en/latest/) relies on MySQL,
+        but a working MySQL can be easily instantiated by Docker and minimal
+        data is seeded by Keyrock - the database volume can be persisted.
+        Knowledge of SQL is unnecessary for use.
+    -   An [IoT Agent](https://iotagent-node-lib.rtfd.io/) can have a dependency
+        on MongoDB - the volumes are exposed so that the data will be persisted
+        if the Docker container goes down. Alternatively an IoT Agent can be
+        memory based, but all configuration can be set up using a config file or
+        Docker params and then data injected by HTTP requests. An understanding
+        of MongoDB is unnecessary to get it to work.
+    -   [QuantumLeap](https://quantumleap.rtfd.io/) works with Grafana and
+        Crate-DB - appropriate set-up and docker compose are available in the
+        Quick Start. Quantum Leap can be used without in depth knowledge of
+        Crate-DB
+
+-   For all webservice components, a `Dockerfile` **MUST** be available within
```

Comment:
a link from `Dockerfile` to https://docs.docker.com/engine/reference/builder/ would be nice